src/Application/Components/Store/Store.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import StoreItem from "./StoreItem";
+
+const Store = React.createClass({
+  propTypes: {
+    assets: React.PropTypes.array.isRequired,
+    createOnClick: React.PropTypes.func.isRequired,
+  },
+
+  componentDidMount() {
+    this.props.assets[0].unlock();
+    this.forceUpdate();
+  },
+
+  render() {
+    let unlockedAssets = [];
+    for (const asset of this.props.assets) {
+      if (asset.unlocked) {
+        unlockedAssets.push(asset);
+      }
+    }
```

Comment:
I'm going to use the underscore method, but I'm going to hold off on the immutable for right now, it will be easier in the next commit when I do the Redux stuff.